src/main/kotlin/net/atos/zac/app/documentcreation/converter/RestDocumentCreationConverter.kt

```diff
@@ -0,0 +1,57 @@
+package net.atos.zac.app.documentcreation.converter
+
+import jakarta.enterprise.inject.Instance
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+import net.atos.client.zgw.drc.decodedBase64StringLength
+import net.atos.client.zgw.drc.model.generated.EnkelvoudigInformatieObjectCreateLockRequest
+import net.atos.client.zgw.drc.model.generated.StatusEnum
+import net.atos.client.zgw.drc.model.generated.VertrouwelijkheidaanduidingEnum
+import net.atos.client.zgw.zrc.model.Zaak
+import net.atos.client.zgw.ztc.ZtcClientService
+import net.atos.zac.app.documentcreation.model.RestDocumentCreationUnattendedData
+import net.atos.zac.authentication.LoggedInUser
+import net.atos.zac.configuratie.ConfiguratieService
+import net.atos.zac.documentcreation.model.DocumentCreationUnattendedResponse
+import net.atos.zac.identity.model.getFullName
+import net.atos.zac.smartdocuments.SmartDocumentsTemplatesService
+import net.atos.zac.util.UriUtil.uuidFromURI
+import nl.lifely.zac.util.AllOpen
+import nl.lifely.zac.util.NoArgConstructor
+import java.time.LocalDate
+
+@Singleton
```

Comment:
I notice that a lot of our services are @Singleton 's but also a lot of then are @ApplicationScoped. Not a big deal but we should make this more consistent at some point probably. Probably they can all be singletons anyway I'm guessing..

Reply:
Yes, true. Or perhaps we can add our own annotation combining these two
Will add `@ApplicationScoped` too